flocker/node/agents/ebs.py

```diff
@@ -1309,13 +1323,6 @@ def get_device_path(self, blockdevice_id):
 
         return _expected_device(ebs_volume.attachments[0]['Device'])
 
-    # ICloudAPI:
-    @boto3_log
-    def list_live_nodes(self):
-        instances = self.connection.instances.filter(
-            Filters=[{'Name': 'instance-state-name', 'Values': ['running']}])
-        return list(instance.id for instance in instances)
-
```

Comment:
Are the `ICloudAPI` removals part of the fix? From here, they look like a bad merge.